src/index.js

```diff
@@ -242,11 +245,20 @@
    * @private
    * @returns {String} Time in H:mm format with am/pm, e.g. '9:30am'
    */
-  function hmmAmPm(date) {
+  function hmmAmPm(date,timeFormat) {
```

Comment:
There is a `[hhmm](https://github.com/artsy/day-schedule-selector/blob/d6a4a5a897ec7ca113b5606bc18ef298974615c8/src/index.js#L257)` helper does what you need here, I believe.

Instead of the `timeFormat` options, IMHO it will be even better if we provide a `formatTimeLabel` option which is a function users can specify that will take a `Date` object and return whatever they want to format it. If the option is not specified, simply use the default `hmmAmPm(d)`.

How do you think about this?